### ISSUE TITLE
Model: Location 

### ISSUE THREAD

**Issue by annabelleee** [archesproject/ARM_Working_Group#15]
![place_20180509](https://user-images.githubusercontent.com/16655095/40398599-ce652cf6-5ded-11e8-9547-a464459c2b13.JPG)

Basic Location Model 

Steps to completion (updated):
- [ ] Metadata
- [ ] Diagram (using ARM WG notation)
- [ ] Sample Data
- [ ] Sample Concepts
- [ ] JSON of Arches branch based on diagram in ARM WG github repo
- [ ] Vote in comments to officially approve Model

**Comment by azaroth42** [comment]
Place Model:
![place_model](https://user-images.githubusercontent.com/871868/41429557-496dca08-6fc2-11e8-9114-ae5fa442c32e.png)

Noting that the SP4 / SP5 part would be a good thing to attempt to create in Arches to see if it works in practice. The pattern relies on SP5 being a Space Primitive, which (one can argue) is an error in the document as the Primitives are typically literals rather than resources.

**Comment by Habennin** [comment]
So this one is also in the google sheet. I agree that the SP5 node and primitive statement seems suspicious. To be thought about.

https://docs.google.com/spreadsheets/d/1EYk1yhhBNWrKbVB0jV_1j8NJ-Egi5zWha7QrVvYvw3s/edit?usp=sharing

It seems to me that the Geometry node is complicated enough to ask for another branch. So I added that in the excel. What do you mean by format and do you have a suggested property for it? (CRM or no?)

**Comment by azaroth42** [comment]
I've brought up the SP5 / E94 issue on the SIG list: http://lists.ics.forth.gr/pipermail/crm-sig/2018-July/003403.html   I anticipate that E94 will be removed from SP5, sending us back to the drawing board.

format -- dc:format, range xsd:string, value should be an iana registered media type.  (e.g. it's a digital, dereferencable information object)

**Comment by azaroth42** [comment]
And to report on the Arches side of things ... it doesn't work without editing the ontology files, as E94 isn't a super class of SP5.

**Comment by azaroth42** [comment]
I propose to use P1_is_identified_by between the Place and the SP5. This would follow the Appellation route (SP5 is a subClass of E47, is a subClass of E44, is a subClass of E41 Appellation).

That gets us out of the Literal vs Resource conundrum, and while the relationship isn't as clear as P168, it's still pretty good -- the geometry can be used to describe the boundary in physical space that provides an objective identity to the place.  (This is I believe that Martin means when he says that there needs to be a discussion between Appellation and Space-time Volume [STV])

**Comment by Habennin** [comment]
This is definitely a possible way to go and basically follows the old CRM strategy of treating the coordinates as a name. We have put CRMgeo people on to the problem though so maybe we can get a final idea/solution on p168 from that direction. In the meantime, I would also support the name route.

**Comment by azaroth42** [comment]
Yup, let's pencil in P1 for now and if there's a better solution emerging, we can switch to it at that point.

**Comment by azaroth42** [comment]
From August meeting discussion:

![2018-08-22 11 29 34](https://user-images.githubusercontent.com/871868/44937704-6be46b00-ad2f-11e8-9ded-5069c503aad7.jpg)

**Comment by annabelleee** [comment]
Change name to Place and now two versions: one simple branch and one simple model. URL to come.
